### PR TITLE
Create new Gov yamls; move old yamls to /notready

### PR DESCRIPTION
**Pull request by OdeclasV**
Create: 

- Gov of Colorado (ID: SCOG21f28540): contact form is very similar to former governor's. I can confirm that thank you message is also the same. It looks like this, https://s3.amazonaws.com/yamltron-dev/screenshots/final/dd4b01ea-c91d-44ed-bad9-078bf3677018.png . Note that on Chrome, success screenshot looks blank but on Firefox it looks fine. 
- Gov of Ohio (ID: SOHG1fc662bc): notice how the success msg looks on this form at the bottom, https://s3.amazonaws.com/yamltron-dev/screenshots/final/dea073b6-f26a-4157-9232-362d1376d2b8.png . It appears as soon as the submit button is clicked and lasts for a few seconds before it disappears again. I added a steps that looks for this specific element in the page when it's available. Tests passed, but it'd be great to know if you have any advice on how to do this more fail-safe Tom. 

Move: 
- state_co_gov_john_hickenlooper
- state_ct_gov_dan_malloy
- state_il_gov_bruce_rauner
- state_oh_gov_john_kasich

### CODE REVIEW

File: members/SOHG1fc662bc.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          value: $ADDRESS_STATE_FULL
+          required: true
+          options: Ohio
+    - click_on:
+        - value: Submit
+          selector: div.col-sm-12 button.margin-bottom-xs
+    - wait:
+        - value: 1
+    - find:
+        - selector: div.odx-message__success-4
```
Comment (tomdionesotes) [inline members/SOHG1fc662bc.yaml]: Have you tried submitting an empty form and made sure we don't find the text on this hidden success message element?

**Comment by tomdionesotes** [comment]
No contact method yet for incoming CT and IL guvs?

File: members/SOHG1fc662bc.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/SOHG1fc662bc.yaml]: Yeah. I did a few tests: with an empty form https://s3.amazonaws.com/yamltron-dev/screenshots/error/c011504d-7f5b-43e4-9a82-e11b79779e6e.png

and a form with some inputs filled out, https://s3.amazonaws.com/yamltron-dev/screenshots/error/4f848510-0417-485c-a03d-372098046d88.png

and the hidden messages deos not appear

**Comment by OdeclasV** [comment]
Nah, no contact form. Fortunately they both have emails

**Pull request merged**